mandatory/2-function-creation.js

```diff
@@ -16,7 +22,16 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (typeof num === "number" && num <= 100) {
```

Comment:
This is an interesting approach, you did very well.

I wonder Shima, would it be good to check if the number is more than 100 first then return `100%` as a string?